src/PhoneValidation.tsx

```diff
@@ -0,0 +1,73 @@
+import type { ChangeEvent } from 'react'
```

Comment:
This is a whole screen or page, meaning that this component covers the entire screen. I would recommend placing it inside `/src/pages`